PusherServer/Pusher.cs

```diff
@@ -192,6 +192,65 @@ public void TriggerAsync(Event[] events, Action<ITriggerResult> callback)
             });
         }
 
+        /// <inheritDoc/>
+        public INotifyResult Notify(string[] interests, string alertText, string alertTitle, string alertSubtitle, string webhook_url, WebhookLevel webhook_level)

```

Comment:
This version of the code is still using RestSharp, we'll need to move it across to the reworked version